### PR TITLE
A small ear related PR (Deer ears, Rabbit ears)

### PR DESCRIPTION
**Pull request by WanderingFox95**
## About The Pull Request

So first of all - the "ears" showing up when you select "deer" in the customization screen are antlers. Literally why?
I fixed that and made some proper deer ears instead. Also while I was on that, I added some straight rabbit ears cause not all rabbits are lop-eared. In fact, lop-eared rabbits don't exist in the wild. Straight ears allow rabbits to hear better and actually notice and evade predators! Lop-ears are a creation of breeders who I guess wanted to make rabbits cuter. Sadly, they're also more prone to getting sick. ...Anyway, I think I digress here, but it's a fact that straight eared rabbits are actually more common. Previews of both additions down below.

Deer ears: 

Straight Rabbit ears:

I even added the ability to change the colour of the ear tips! (You may or may not need to use advanced colouring for correct application, depending on the rest of your markings though)

## Why It's Good For The Game

Antlers are literally not ears. Also, not all rabbits are lop-eared. That is all.

## Changelog
:cl:
imageadd: Replaced the antlers showing up when you select deer ears with actual deer ears. Literally why was that even a thing before?
imageadd: Straight rabbit ears are now a thing.
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 1]
```diff
@@ -162,12 +162,19 @@
 	icon = 'modular_citadel/icons/mob/mam_ears.dmi'
 
 /datum/sprite_accessory/ears/human/rabbit
-	name = "Rabbit"
+	name = "Rabbit(lop-eared)"
```
Comment (zeroisthebiggay) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm]:
```suggestion
	name = "Rabbit (Lop-eared)"
```
inconsistent capitalization and spacing

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 2]
```diff
@@ -162,12 +162,19 @@
...
 	icon_state = "rabbit"
 	color_src = MATRIXED
 	matrixed_sections = MATRIX_RED_GREEN
 	icon = 'modular_citadel/icons/mob/mam_ears.dmi'
 
+/datum/sprite_accessory/ears/human/rabbitalt
+	name = "Rabbit"
```
Comment (zeroisthebiggay) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm]:
```suggestion
	name = "Rabbit (Straight-eared)"
```
if you are gonna tag lops then you should tag this too

**Comment by WanderingFox95** [comment]
For consistency, I applied your suggestions in the anthro section too.

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 3]
```diff
@@ -359,10 +366,15 @@
 	matrixed_sections = MATRIX_RED_GREEN
 
 /datum/sprite_accessory/ears/mam_ears/rabbit
-	name = "Rabbit"
+	name = "Rabbit (lop-eared)"
```
Comment (zeroisthebiggay) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm]:
```suggestion
	name = "Rabbit (Lop-eared)"
```
inconsistent capitalization

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 4]
```diff
@@ -359,10 +366,15 @@
...
 	icon_state = "rabbit"
 	matrixed_sections = MATRIX_RED_GREEN
 
+/datum/sprite_accessory/ears/mam_ears/rabbitalt
+	name = "Rabbit (straight-eared)"
```
Comment (zeroisthebiggay) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm]:
```suggestion
	name = "Rabbit (Straight-eared)"
```
inconsistent capitalization

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm (hunk 3 above)
Comment (WanderingFox95) [reply code/modules/mob/dead/new_player/sprite_accessories/ears.dm]: also fair. though capitalization is very inconsistent throughout the entire list. Skunk and Kangaroo aren't capitalized, for instance.

**Comment by WanderingFox95** [comment]
Right, consistency achieved.

**Pull request merged**